server/fakedb/Schedule.csv

```diff
@@ -0,0 +1,22 @@
+﻿eventId,start,end,locationLatitude,locationLongitude,repeat
```

Comment:
Why is the lat lng stored instead of actual address? Computing from address to lat lng is easier compared to doing the reverse.